draft-ietf-tls-dtls13.md

```diff
@@ -956,6 +957,10 @@ configured not to perform a cookie exchange.  The default SHOULD be
 that the exchange is performed, however.  In addition, the server MAY
 choose not to do a cookie exchange when a session is resumed or, more
 generically, when the DTLS handshake uses a PSK-based key exchange.
+Servers which process 0-RTT requests and send 0.5-RTT responses
+without a cookie exchange should
```

Comment:
SHOULD?

Isn't the alternative here to acknowledge the possibility that 0.5-RTT response are sent to an address that has not demonstrated a willingness to communicate.

Reply:
Do you have proposed text.